src/services/sections/index.js

```diff
@@ -1,18 +1,28 @@
 const { SectionServiceHooks, SectionService } = require('./section.service');
 const { SectionDiffService } = require('./sectionDiff.service');
 const { SectionModelService } = require('./SectionModel.service');
+const { publishData } = require('../../utils/sockets');
 
 module.exports = (app) => {
 	app.configure(SectionModelService);
 
 	const path = 'lesson/:lessonId/sections';
-	const diffSuffix = '/diff';
+	const diffPath = `${path}/diff`;
 
 	app.use(path, new SectionService({}));
-	app.use(path + diffSuffix, new SectionDiffService({
+
+	const sectionService = app.service(path);
+	sectionService.publish('created', publishData(app, 'sections'));
```

Comment:
Should it be added at this point, or as app.configure(socketSetup) with all related sockets actions like joinLessonChannel  publish and so on.